### PR TITLE
fix: 🐛 Fail gracefully on an MPR load error

### PR DESCRIPTION
**Pull request by JamesAPetts**
RE: #1986.

Related: https://github.com/OHIF/react-vtkjs-viewport/pull/107

- Display an error notification when one or more images fail to return from the server.
- Gracefully fail and complete loading when all promises are processed, regardless if they were successful or not. This allows us to still view an image which had few problematic requests, e.g. a single missing/malformed frame.

If many requests fail it will only show the error for the first one, to reduce spam.

Example below. I just disconnected my internet here so there is no error message, but if your server returns an error message it will be shown in the notfication:

FWIW with #1984 in place, I couldn't reproduce "real" errors on the IDC proxy with multiple tabs, which is good. I have tested faked error messages work correctly.

### CODE REVIEW

File: extensions/vtk/src/OHIFVTKViewport.js [hunk 1]
```diff
@@ -66,6 +68,7 @@ class OHIFVTKViewport extends Component {
     viewportIndex: PropTypes.number,
     children: PropTypes.node,
     onScroll: PropTypes.func,
+    servicesManager: PropTypes.object,
```
Comment (dannyrb) [inline extensions/vtk/src/OHIFVTKViewport.js:71]: Required? In general, prefer `required` as default unless there's a good reason to mark as optional. Optional means you have one more edge case to guard/test.

**Comment by JamesAPetts** [comment]
The `servicesManager` is just being passed into the wrapper so that we can use the `UINotificationService` on errors, not being passed to the 3rd party lib. I'll update the branch and merge then.
